src/xlsx/conversion/figure_conversion.ts

```diff
@@ -47,15 +51,18 @@ function convertFigure(
 }
 
 function convertChartData(chartData: ExcelChartDefinition): ChartDefinition | undefined {
-  const labelRange = chartData.dataSets[0].label?.replace(/\$/g, "");
-  let dataSets = chartData.dataSets.map((data) => data.range.replace(/\$/g, ""));
+  const dataSetsHaveTitle = chartData.dataSets[0].label !== undefined;
+  const labelRange = convertExcelLabelToSheetXC(chartData.labelRange, dataSetsHaveTitle);
```

Comment:
@LucasLefevre Because now we exclude the whole row rather than a cell when we ticked `dataset has title`, it will also influence the exporting .xlsx file. And also when we import .xlsx file we also want to automatically tick "use row x as headers" if applicable, so part of code are also about it.

Reply:
for the record, we are already converting excel stuff in the function (if you look at the function arguments)

we could argue that those helpers `convertExcelDatasetToSheetXC` and `convertExcelLabelToSheetXC` don't belong in chart common - they should be defined in the place where they are used -`figure_conversion.ts`. 

FUrthermore, both helpers seem to do the exact same thing (we only process the range of the ExcelChartdataset, and in the same way we do with the label) So it'd be nice to have a single function `convertExcelRangeToSHeetXC` or have both functions call a common helper.